go.mod

```diff
@@ -1,14 +1,14 @@
 module github.com/cli/cli/v2
 
-go 1.22
+go 1.22.0
```

Comment:
It's quite irritating that we are being forced to change this because `sigstore-go` was forced to, because `sigstore` was forced to because `letsencrypt/boulder` unnecessarily used `1.22.0` instead of `1.22` to get `range-over-int` (at least, that appears to be the chain). Now _our_ consumers (if we have any) are forced to bump as well.

Unfortunately I don't see any way around it right now. Perhaps we can communicate upstream for when they change in future.